packages/next/build/webpack-config.ts

```diff
@@ -485,8 +481,7 @@ export default async function getBaseWebpackConfig(
             if (
               !res.match(/next[/\\]dist[/\\]next-server[/\\]/) &&
               (res.match(/[/\\]next[/\\]dist[/\\]/) ||
-                res.match(/node_modules[/\\]@babel[/\\]runtime[/\\]/) ||
-                res.match(/node_modules[/\\]@babel[/\\]runtime-corejs2[/\\]/))
+                res.match(/node_modules[/\\]@babel[/\\]runtime[/\\]/))
```

Comment:
Let's maybe comment that _this_ runtime is Babel's `helpers: true` option (if this is true, I dunno).